modules/graph_ql/main.tf

```diff
@@ -1,4 +1,133 @@
-data "aws_iam_policy_document" "graphql_task_execution_policy_document" {
+# Local Variables
+locals {
+  graphql_port = 5000
```

Comment:
Our Dockerfile needs a few tweaks to make this truly configurable. Created https://bugs.earthdata.nasa.gov/browse/ORCA-581